frontend/document/credocument.lua

```diff
@@ -1494,10 +1494,27 @@ function CreDocument:buildSyntheticPageMapIfNoneDocumentProvided(chars_per_synth
     self._document:buildSyntheticPageMapIfNoneDocumentProvided(chars_per_synthetic_page or 1024)
 end
```

Comment:
Didn't you want to emulate/rewrite that one fully in frontend using the new cre methods only?